include/mbedtls/dhm.h

```diff
@@ -92,8 +111,18 @@
     "BBE117577A615D6C770988C0BAD946E208E24FA074E5AB31" \
     "43DB5BFCE0FD108E4B82D120A93AD2CAFFFFFFFFFFFFFFFF"
 
+/** 
+ * The generator of the 3072-bit MODP Group, as defined in 
```

Comment:
I see that you haven't yet changed this; my updated suggestion for all groups would be `The hexadecimal presentation of [the chosen generator of | the prime underlying the] XXX-bit MODP Group, as defined in XXX`.

Reply:
Standardized in all